<launch>
    <node pkg="euclidean_cluster" type="euclidean_cluster_node" name="euclidean_cluster_node" output="screen">
        <param name="lidar_height" value="0.34" type="double"/>
        <param name="down_height" value="0.1" type="double"/>
        <param name="up_height" value="0.7" type="double"/>
        <param name="low_x" value="0.2" type="double"/>
        <param name="high_x" value="2.0" type="double"/>
        <param name="low_y" value="-0.5" type="double"/>
        <param name="high_y" value="0.5" type="double"/>
        <param name="in_cloud_topic" value="/velodyne_points" type="string" />
    </node>
</launch>
